egs2/TEMPLATE/enh_diar1/enh_diar.sh

```diff
@@ -0,0 +1,905 @@
+                    "${data_feats}${_suf}/${dset}"/segments \
+                    "${data_feats}${_suf}/${dset}"/rttm
+
+            # convert standard rttm file into espnet-format rttm (measure with samples)
+            pyscripts/utils/convert_rttm.py \
+                --rttm "${data_feats}${_suf}/${dset}"/rttm \
+                --wavscp "${data_feats}${_suf}/${dset}"/wav.scp \
+                --output_path "${data_feats}${_suf}/${dset}" \
+                --sampling_rate "${fs}"
+        done
+    fi
+
+
+    if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
+        log "Stage 3: Remove short data: ${data_feats}/org -> ${data_feats}"
+
+        for dset in "${train_set}" "${valid_set}"; do
+        # NOTE: Not applying to test_sets to keep original data
+
+            _spk_list=" "
+            _scp_list=" "
+            for i in $(seq ${spk_num}); do
+                _spk_list+="spk${i} "
+                _scp_list+="spk${i}.scp "
+            done
+            if $use_noise_ref; then
+                # references for denoising ("noise1 noise2 ... niose${noise_type_num} ")
+                _spk_list+=$(for n in $(seq $noise_type_num); do echo -n "noise$n "; done)
+                _scp_list+=$(for n in $(seq $noise_type_num); do echo -n "noise$n.scp "; done)
+            fi
+            if $use_dereverb_ref; then
+                # references for dereverberation
+                _spk_list+=$(for n in $(seq $dereverb_ref_num); do echo -n "dereverb$n "; done)
+                _scp_list+=$(for n in $(seq $dereverb_ref_num); do echo -n "dereverb$n.scp "; done)
+            fi
+
+            # Copy data dir
+            utils/copy_data_dir.sh "${data_feats}/org/${dset}" "${data_feats}/${dset}"
+            cp "${data_feats}/org/${dset}/feats_type" "${data_feats}/${dset}/feats_type"
+            for spk in ${_spk_list};do
+                cp "${data_feats}/org/${dset}/${spk}.scp" "${data_feats}/${dset}/${spk}.scp"
+            done
+
+            _fs=$(python3 -c "import humanfriendly as h;print(h.parse_size('${fs}'))")
+            _min_length=$(python3 -c "print(int(${min_wav_duration} * ${_fs}))")
+
+            # utt2num_samples is created by format_wav_scp.sh
+            # diarization typically accept long recordings, so does not has
+            # max length requirements
+            <"${data_feats}/org/${dset}/utt2num_samples" \
+                awk -v min_length="${_min_length}" \
+                    '{ if ($2 > min_length ) print $0; }' \
+                    >"${data_feats}/${dset}/utt2num_samples"
+            for spk in ${_spk_list} "wav"; do
+                <"${data_feats}/org/${dset}/${spk}.scp" \
+                    utils/filter_scp.pl "${data_feats}/${dset}/utt2num_samples"  \
+                    >"${data_feats}/${dset}/${spk}.scp"
+            done
+
+            # fix_data_dir.sh leaves only utts which exist in all files
+            #utils/fix_data_dir.sh --utt_extra_files "${_scp_list}" "${data_feats}/${dset}"
```

Comment:
Can you remove the commented lines if they are not used?

Reply:
removed them